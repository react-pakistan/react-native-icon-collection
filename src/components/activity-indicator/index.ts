export {
  ActivityIndicator,
  ActivityIndicatorSize,
} from './activity-indicator';
