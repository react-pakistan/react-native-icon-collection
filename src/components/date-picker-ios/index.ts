export { DatePickerIOS } from './date-picker-ios';
