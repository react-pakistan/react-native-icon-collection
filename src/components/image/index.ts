export {
  IImageProps,
  Image,
  ResizeMethod,
  ResizeMode,
} from './image';
