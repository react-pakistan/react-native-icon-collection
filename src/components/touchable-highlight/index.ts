export { TouchableHighlight } from './touchable-highlight';
