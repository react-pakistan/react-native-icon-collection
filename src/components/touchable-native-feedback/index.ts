export { TouchableNativeFeedback } from './touchable-native-feedback';
