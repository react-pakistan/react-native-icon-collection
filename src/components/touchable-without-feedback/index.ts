export { TouchableWithoutFeedback } from './touchable-without-feedback';
