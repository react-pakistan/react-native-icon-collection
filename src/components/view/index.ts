export { View, IView } from './view';
