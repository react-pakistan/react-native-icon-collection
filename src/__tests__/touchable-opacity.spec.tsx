import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';
import { TouchableOpacity } from '../components/touchable-opacity';
// import {
// } from '../story/view.story';

describe('Testing TouchableOpacity component', () => {
  it('matches TouchableOpacity snapshot', () => {
    const wrapper = renderer.create(
      <TouchableOpacity>
        <Text>
          Hello World
        </Text>
      </TouchableOpacity>,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
