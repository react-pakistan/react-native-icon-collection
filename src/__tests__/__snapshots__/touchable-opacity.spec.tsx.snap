// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing TouchableOpacity component matches TouchableOpacity snapshot 1`] = `
<View
  accessible={true}
  focusable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <Text>
    Hello World
  </Text>
</View>
`;
